Source/Core/VideoCommon/OnScreenDisplay.cpp

```diff
@@ -48,8 +50,8 @@ struct Message
   bool ever_drawn = false;
   bool should_discard = false;
   u32 color = 0;
-  std::unique_ptr<Icon> icon;
-  std::unique_ptr<AbstractTexture> texture;
+  std::shared_ptr<Icon> icon;
+  std::shared_ptr<AbstractTexture> texture;
```

Comment:
Why did you change this, what exactly are you trying to accomplish by doing so? Same for all other changes in this file.

Reply:
It's a pointer I use in multiple places, I can't copy the unique_ptr and I can't move it because I might need it again, what am I misunderstanding about how this works?

Reply:
Never mind, your other comment about why I'm not decoding when I need it resolves this.